src/Css.elm

```diff
@@ -1454,20 +1556,20 @@ introduceSelector selector declarations =
             [ StyleBlock (SingleSelector selector) [] [] ]
 
         {- If there are no properties declared, then we have a situation
-            like `Bar` in the following:
+                            like `Bar` in the following:
 
-                stylesheet "homepage"
-                    . Foo . Bar
-                        ~ fontWeight bold
+                                stylesheet "homepage"
+                                    . Foo . Bar
+                                        ~ fontWeight bold
 
-                ...as opposed to:
+                                ...as opposed to:
 
-                stylesheet "homepage"
-                    . Foo
-                        ~ fontWeight bold
+                                stylesheet "homepage"
+                                    . Foo
+                                        ~ fontWeight bold
 
-            In this case, we don't make a new declaration, but rather add
-            the new selector to the existing list of selectors.
+                            In this case, we don't make a new declaration, but rather add
+                            the new selector to the existing list of selectors.
```

Comment:
Aha, looks like you encountered https://github.com/avh4/elm-format/issues/90 here :wink: - I've been deliberately excluding these from my commits, but maybe I should just switch this to use `--` comments until it's fixed...